pkg/shares/README.md

```diff
@@ -0,0 +1,82 @@
+# Shares
+
+Package shares provides primitives for splitting block data into shares and
+parsing shares back into block data.
```

Comment:
[Non Blocking][Suggestion]
I guess that editors nowadays handle soft line wrapping perfectly. Thus, no need to add new lines in the middle of a sentence.